projects/igniteui-angular/src/lib/grids/state.directive.spec.ts

```diff
@@ -389,14 +389,42 @@ describe('IgxGridState - input properties #grid', () => {
         let gridState = state.getState(true, 'columns');
         expect(gridState).toBe(initialState);
 
-        state.setState(columnsStateObject);
+        state.setState(columnsStateObject); // set new state
         gridState = state.getState(false, 'columns') as IGridState;
         HelperFunctions.verifyColumns(columnsStateObject.columns, gridState);
         gridState = state.getState(true, 'columns');
         expect(gridState).toBe(columnsState);
         expect(grid.columnInit.emit).toHaveBeenCalledTimes(columnsStateObject.columns.length);
     });
 
+    it('setState should correctly restore grid columns state properties: collapsible and expanded', () => {
+        const fix = TestBed.createComponent(CollapsibleColumnGroupTestComponent);
+        fix.detectChanges();
+        const state = fix.componentInstance.state;
+        const grid = fix.componentInstance.grid;
+        spyOn(grid.columnInit, 'emit').and.callThrough();
```

Comment:
The test is not about verifying that the `columnInit` event was emitted the expected number of times, so this ca be removed.